test/e2e/framework/metrics_util.go

```diff
@@ -233,60 +233,27 @@ func (l *SchedulingMetrics) PrintJSON() string {
 	return PrettyPrintJSON(l)
 }
 
-type histogram struct {
-	Count   int
-	Buckets map[float64]int
+type Histogram struct {
+	Labels  map[string]string `json:"labels"`
+	Buckets map[string]int    `json:"buckets"`
```

Comment:
Maybe `BucketCounts`? (as this is not just buckets)

Reply:
I would like to keep `buckets` name. I think term `bucket` is often used to describe something that has label and count of objects which this label apply to.

Reply:
If you're particular, I'm fine with it. But, that doesn't really match with for e.g our prometheus convention. See - https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/prometheus/client_golang/prometheus/histogram.go#L153 (for buckets) and https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/prometheus/client_golang/prometheus/histogram.go#L231 (for counts). Since you're storing both of them together, I suggested BucketCounts.

Reply:
In context of histogram, those "`buckets`" from options structure are just `upperBounds`.
Here is a example of bucket structure https://github.com/prometheus/prometheus/blob/f32acc0b7b2bbe641d573a9abdfa96f600d01073/vendor/github.com/prometheus/client_golang/prometheus/histogram.go#L266

Reply:
Ok, I can live with this name :)